spec/features/user_views_prs_spec.rb

```diff
@@ -116,9 +116,9 @@
       click_on "rails"
     end
 
-    expect(page).to have_content("An Ember PR")
-    expect(page).to have_content("A Rails PR")
-    expect(page).to have_content("A long pr")
+    expect(page).to have_content("An Ember PR", count: 1)
+    expect(page).to have_content("A Rails PR", count: 1)
+    expect(page).to have_content("A long pr", count: 1)
```

Comment:
Either one is fine. i don't usually care for `have_content` in general, it's not specific enough but I think this is fine.